requirements.txt

```diff
@@ -0,0 +1,3 @@
+osmeterium==0.0.3
+mapcoloniesjsonlogger==1.0.0
+pytest
```

Comment:
this file should be deleted since we use pipenv

we can create requirements.txt by running
```sh
pipenv run pip freeze > requirements.txt
```

please add `requirements.txt` to `.gitignore`  and remove it from the PR

Reply:
done.